src/CBot/CBotUtils.cpp

```diff
@@ -245,6 +245,12 @@ bool CharInList(const char c, const char* list)
 
 std::string CodePointToUTF8(unsigned int val)
 {
+    // nota : similare with StrUtils::UnicodeCharToUtf8 (with traces but CBotErrUnicodeName)
+    //  memo UTF8:
```

Comment:
Eh, I'm not sure I want this memo scattered in multiple places but I'm not sure where would be one good place to put it. I think it should just be removed, it is obvious from the name of the function that anyone interested of how the implementation works should read the UTF-8 specification.